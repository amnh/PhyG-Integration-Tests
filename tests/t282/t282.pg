--build(graph:hardwired), inputting tcm and nucleotide files
set(seed:1634561640)
set(graphtype:hardwired)
read(nucleotide:"chel-prealigned.fas", tcm:"sg1t4.mat")
read(nucleotide:"chel_cox1aln.fasta", tcm:"sg2t1.mat")
build(distance, rdwag, block, eun, displaytrees:10, atrandom, graph)
report("t282.csv", data, overwrite)
report("t282.tre", newick, overwrite)
report("t282.dot", dotpdf, overwrite)