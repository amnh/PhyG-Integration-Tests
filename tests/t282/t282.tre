(Alentus:0.0,(Chanbria:136.0,((((Artemia:132.0,Thermobius:140.0)HTU30:38.0,#HTU34:0.0)HTU26:12.0,(Rhiphicephalus:152.0,(Vonones:84.0,(Pauroctonus:83.0,((Mastigoproctus:98.0,(Hypochilus:126.0,Thelichoris:104.0)HTU29:56.0)HTU32:20.0,#HTU35:0.0)HTU24:15.0)HTU27:14.0)HTU31:14.0)HTU28:51.0)HTU25:2.0,(((Amblypygid:76.0,(Americhernus:147.0,Aportus:120.0)HTU20:65.0)HTU21:72.0,(Limulus:0.0)#HTU35:0.0)HTU18:34.0,((Centruoides:90.0,Hadrurus:112.0)HTU22:76.0,(Gea:0.0)#HTU34:0.0)HTU19:12.0)HTU36:4.0)HTU23:43.0)HTU33:0.0)HTU17[2549.0];
(Alentus:183.0,(Chanbria:138.0,(((Gea:113.0,(Artemia:126.0,Thermobius:135.0)HTU28:44.0)HTU24:25.0,(Rhiphicephalus:130.0,(Vonones:80.0,(Pauroctonus:67.0,(Mastigoproctus:68.0,(Hypochilus:118.0,Thelichoris:120.0)HTU27:73.0)HTU30:20.0)HTU25:18.0)HTU29:25.0)HTU26:26.0)HTU23:8.0,((Limulus:146.0,(Amblypygid:89.0,(Americhernus:186.0,Aportus:111.0)HTU19:76.0)HTU20:50.0)HTU18:18.0,(Centruoides:98.0,Hadrurus:116.0)HTU21:87.0)HTU32:16.0)HTU22:20.0)HTU31:83.0)HTU17[2289.0];
(Alentus:183.0,(Chanbria:138.0,(((Gea:113.0,(Artemia:126.0,Thermobius:135.0)HTU28:44.0)HTU24:25.0,(Rhiphicephalus:130.0,(Vonones:80.0,(Pauroctonus:67.0,(Mastigoproctus:68.0,(Hypochilus:118.0,Thelichoris:120.0)HTU27:73.0)HTU30:20.0)HTU25:18.0)HTU29:25.0)HTU26:26.0)HTU23:8.0,((Limulus:146.0,(Amblypygid:89.0,(Americhernus:186.0,Aportus:111.0)HTU19:76.0)HTU20:50.0)HTU18:18.0,(Centruoides:98.0,Hadrurus:116.0)HTU21:87.0)HTU32:16.0)HTU22:20.0)HTU31:83.0)HTU17[2289.0];
(Alentus:167.0,(Chanbria:140.0,(((Gea:127.0,(Artemia:123.0,Thermobius:134.0)HTU28:37.0)HTU24:27.0,(Rhiphicephalus:137.0,(Vonones:96.0,(Pauroctonus:84.0,(Limulus:118.0,(Mastigoproctus:98.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU22:15.0)HTU25:18.0)HTU29:11.0)HTU26:41.0)HTU23:6.0,((Amblypygid:70.0,(Americhernus:201.0,Aportus:107.0)HTU18:68.0)HTU19:115.0,(Centruoides:83.0,Hadrurus:123.0)HTU20:78.0)HTU32:9.0)HTU21:24.0)HTU31:56.0)HTU17[2287.0];
(Alentus:167.0,(Chanbria:140.0,(((Gea:127.0,(Artemia:123.0,Thermobius:134.0)HTU28:37.0)HTU24:27.0,(Rhiphicephalus:137.0,(Vonones:96.0,(Pauroctonus:84.0,(Limulus:118.0,(Mastigoproctus:98.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU22:15.0)HTU25:18.0)HTU29:11.0)HTU26:41.0)HTU23:6.0,((Amblypygid:70.0,(Americhernus:201.0,Aportus:107.0)HTU18:68.0)HTU19:115.0,(Centruoides:83.0,Hadrurus:123.0)HTU20:78.0)HTU32:9.0)HTU21:24.0)HTU31:56.0)HTU17[2287.0];
(Alentus:167.0,(Chanbria:140.0,(((Gea:127.0,(Artemia:123.0,Thermobius:134.0)HTU28:37.0)HTU24:27.0,(Rhiphicephalus:137.0,(Vonones:96.0,(Pauroctonus:84.0,(Limulus:118.0,(Mastigoproctus:98.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU22:15.0)HTU25:18.0)HTU29:11.0)HTU26:41.0)HTU23:6.0,((Amblypygid:70.0,(Americhernus:201.0,Aportus:107.0)HTU18:68.0)HTU19:115.0,(Centruoides:83.0,Hadrurus:123.0)HTU20:78.0)HTU32:9.0)HTU21:24.0)HTU31:56.0)HTU17[2287.0];
(Alentus:214.0,(Chanbria:157.0,(((Rhiphicephalus:128.0,(Vonones:91.0,(Pauroctonus:69.0,(Mastigoproctus:69.0,(Hypochilus:130.0,Thelichoris:129.0)HTU27:64.0)HTU30:72.0)HTU25:8.0)HTU29:41.0)HTU26:21.0,(Artemia:156.0,Thermobius:110.0)HTU28:33.0)HTU24:14.0,((Limulus:88.0,(Amblypygid:80.0,(Americhernus:181.0,Aportus:114.0)HTU20:53.0)HTU21:59.0)HTU18:16.0,(Gea:115.0,(Centruoides:93.0,Hadrurus:104.0)HTU22:72.0)HTU19:25.0)HTU32:15.0)HTU23:28.0)HTU31:121.0)HTU17[2273.0];
(Alentus:214.0,(Chanbria:157.0,(((Rhiphicephalus:128.0,(Vonones:91.0,(Pauroctonus:69.0,(Mastigoproctus:69.0,(Hypochilus:130.0,Thelichoris:129.0)HTU27:64.0)HTU30:72.0)HTU25:8.0)HTU29:41.0)HTU26:21.0,(Artemia:156.0,Thermobius:110.0)HTU28:33.0)HTU24:14.0,((Limulus:88.0,(Amblypygid:80.0,(Americhernus:181.0,Aportus:114.0)HTU20:53.0)HTU21:59.0)HTU18:16.0,(Gea:115.0,(Centruoides:93.0,Hadrurus:104.0)HTU22:72.0)HTU19:25.0)HTU32:15.0)HTU23:28.0)HTU31:121.0)HTU17[2273.0];
(Alentus:167.0,(Chanbria:165.0,(((Rhiphicephalus:125.0,(Vonones:93.0,(Pauroctonus:85.0,(Limulus:122.0,(Mastigoproctus:102.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU23:15.0)HTU25:21.0)HTU29:23.0)HTU26:28.0,(Artemia:146.0,Thermobius:100.0)HTU28:63.0)HTU24:10.0,((Gea:123.0,(Centruoides:81.0,Hadrurus:108.0)HTU21:77.0)HTU18:32.0,(Amblypygid:77.0,(Americhernus:183.0,Aportus:114.0)HTU19:85.0)HTU20:47.0)HTU32:23.0)HTU22:20.0)HTU31:49.0)HTU17[2266.0];
(Alentus:167.0,(Chanbria:165.0,(((Rhiphicephalus:125.0,(Vonones:93.0,(Pauroctonus:85.0,(Limulus:122.0,(Mastigoproctus:102.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU23:15.0)HTU25:21.0)HTU29:23.0)HTU26:28.0,(Artemia:146.0,Thermobius:100.0)HTU28:63.0)HTU24:10.0,((Gea:123.0,(Centruoides:81.0,Hadrurus:108.0)HTU21:77.0)HTU18:32.0,(Amblypygid:77.0,(Americhernus:183.0,Aportus:114.0)HTU19:85.0)HTU20:47.0)HTU32:23.0)HTU22:20.0)HTU31:49.0)HTU17[2266.0];
(Alentus:167.0,(Chanbria:165.0,(((Rhiphicephalus:125.0,(Vonones:93.0,(Pauroctonus:85.0,(Limulus:122.0,(Mastigoproctus:102.0,(Hypochilus:126.0,Thelichoris:104.0)HTU27:58.0)HTU30:20.0)HTU23:15.0)HTU25:21.0)HTU29:23.0)HTU26:28.0,(Artemia:146.0,Thermobius:100.0)HTU28:63.0)HTU24:10.0,((Gea:123.0,(Centruoides:81.0,Hadrurus:108.0)HTU21:77.0)HTU18:32.0,(Amblypygid:77.0,(Americhernus:183.0,Aportus:114.0)HTU19:85.0)HTU20:47.0)HTU32:23.0)HTU22:20.0)HTU31:49.0)HTU17[2266.0];
