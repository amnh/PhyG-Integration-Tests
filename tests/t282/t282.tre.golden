(Alentus:99.5,(Chanbria:98.5,((Rhiphicephalus:160.5,(Artemia:171.5,(((Hypochilus:71.5,Thelichoris:103.5)HTU32:41.0,#HTU34:701.5)HTU27:39.5,(Mastigoproctus:113.0,#HTU35:729.5)HTU31:36.5)HTU28:36.5)HTU30:11.0)HTU29:42.5,(((((Vonones:91.5,(Pauroctonus:92.5,Thermobius:147.5)HTU24:48.5)HTU25:25.0,(Limulus:733.0)#HTU35:733.0)HTU19:23.5,(Amblypygid:104.0,(Americhernus:190.5,Aportus:161.5)HTU21:84.0)HTU20:60.5)HTU18:45.5,(Gea:718.5)#HTU34:718.5)HTU22:37.5,(Centruoides:82.0,Hadrurus:114.0)HTU23:74.0)HTU36:42.5)HTU26:37.0)HTU33:111.0)HTU17[1925.0];
(Alentus:103.0,(Chanbria:88.5,((Rhiphicephalus:140.0,(Artemia:182.0,((Limulus:117.0,Mastigoproctus:101.0)HTU29:31.0,(Hypochilus:99.0,Thelichoris:76.0)HTU30:45.0)HTU26:39.0)HTU28:17.0)HTU27:48.5,((Gea:156.5,((Amblypygid:110.0,(Americhernus:188.5,Aportus:163.5)HTU20:79.5)HTU19:52.0,(Vonones:95.0,(Pauroctonus:100.5,Thermobius:146.5)HTU23:45.5)HTU24:34.0)HTU18:60.5)HTU21:51.0,(Centruoides:82.0,Hadrurus:113.0)HTU22:82.5)HTU32:54.5)HTU25:40.0)HTU31:114.5)HTU17[1748.0];
(Alentus:103.0,(Chanbria:88.5,((Rhiphicephalus:140.0,(Artemia:182.0,((Limulus:117.0,Mastigoproctus:101.0)HTU29:31.0,(Hypochilus:99.0,Thelichoris:76.0)HTU30:45.0)HTU26:39.0)HTU28:17.0)HTU27:48.5,((Gea:156.5,((Amblypygid:110.0,(Americhernus:188.5,Aportus:163.5)HTU20:79.5)HTU19:52.0,(Vonones:95.0,(Pauroctonus:100.5,Thermobius:146.5)HTU23:45.5)HTU24:34.0)HTU18:60.5)HTU21:51.0,(Centruoides:82.0,Hadrurus:113.0)HTU22:82.5)HTU32:54.5)HTU25:40.0)HTU31:114.5)HTU17[1748.0];
(Alentus:104.0,(Chanbria:91.5,((Rhiphicephalus:142.5,(Artemia:165.5,(Mastigoproctus:98.5,(Hypochilus:94.5,Thelichoris:73.5)HTU30:54.5)HTU27:43.5)HTU29:31.0)HTU28:50.5,((Gea:160.5,((Limulus:81.5,(Vonones:96.5,(Pauroctonus:94.5,Thermobius:148.5)HTU24:49.5)HTU25:27.0)HTU19:27.5,(Amblypygid:107.0,(Americhernus:189.0,Aportus:169.0)HTU21:87.0)HTU20:67.5)HTU18:50.0)HTU22:48.0,(Centruoides:75.0,Hadrurus:113.0)HTU23:79.5)HTU32:47.0)HTU26:36.5)HTU31:112.0)HTU17[1734.0];
(Alentus:104.0,(Chanbria:91.5,((Rhiphicephalus:142.5,(Artemia:165.5,(Mastigoproctus:98.5,(Hypochilus:94.5,Thelichoris:73.5)HTU30:54.5)HTU27:43.5)HTU29:31.0)HTU28:50.5,((Gea:160.5,((Limulus:81.5,(Vonones:96.5,(Pauroctonus:94.5,Thermobius:148.5)HTU24:49.5)HTU25:27.0)HTU19:27.5,(Amblypygid:107.0,(Americhernus:189.0,Aportus:169.0)HTU21:87.0)HTU20:67.5)HTU18:50.0)HTU22:48.0,(Centruoides:75.0,Hadrurus:113.0)HTU23:79.5)HTU32:47.0)HTU26:36.5)HTU31:112.0)HTU17[1734.0];
(Alentus:104.0,(Chanbria:91.5,((Rhiphicephalus:142.5,(Artemia:165.5,(Mastigoproctus:98.5,(Hypochilus:94.5,Thelichoris:73.5)HTU30:54.5)HTU27:43.5)HTU29:31.0)HTU28:50.5,((Gea:160.5,((Limulus:81.5,(Vonones:96.5,(Pauroctonus:94.5,Thermobius:148.5)HTU24:49.5)HTU25:27.0)HTU19:27.5,(Amblypygid:107.0,(Americhernus:189.0,Aportus:169.0)HTU21:87.0)HTU20:67.5)HTU18:50.0)HTU22:48.0,(Centruoides:75.0,Hadrurus:113.0)HTU23:79.5)HTU32:47.0)HTU26:36.5)HTU31:112.0)HTU17[1734.0];
(Alentus:103.5,(Chanbria:85.0,((Rhiphicephalus:148.5,(Artemia:172.0,((Gea:154.0,(Hypochilus:68.5,Thelichoris:97.5)HTU30:37.0)HTU25:38.0,(Limulus:102.0,Mastigoproctus:109.0)HTU29:35.0)HTU26:43.0)HTU28:20.5)HTU27:37.5,(((Amblypygid:90.5,(Americhernus:194.5,Aportus:142.5)HTU20:81.0)HTU19:50.5,(Vonones:90.0,(Pauroctonus:90.5,Thermobius:145.5)HTU22:49.0)HTU23:34.0)HTU18:50.5,(Centruoides:77.0,Hadrurus:114.0)HTU21:83.0)HTU32:39.5)HTU24:37.5)HTU31:104.5)HTU17[1730.0];
(Alentus:103.5,(Chanbria:85.0,((Rhiphicephalus:148.5,(Artemia:172.0,((Gea:154.0,(Hypochilus:68.5,Thelichoris:97.5)HTU30:37.0)HTU25:38.0,(Limulus:102.0,Mastigoproctus:109.0)HTU29:35.0)HTU26:43.0)HTU28:20.5)HTU27:37.5,(((Amblypygid:90.5,(Americhernus:194.5,Aportus:142.5)HTU20:81.0)HTU19:50.5,(Vonones:90.0,(Pauroctonus:90.5,Thermobius:145.5)HTU22:49.0)HTU23:34.0)HTU18:50.5,(Centruoides:77.0,Hadrurus:114.0)HTU21:83.0)HTU32:39.5)HTU24:37.5)HTU31:104.5)HTU17[1730.0];
(Alentus:106.5,(Chanbria:83.5,((Rhiphicephalus:150.5,(Artemia:164.5,(Mastigoproctus:94.0,(Gea:150.0,(Hypochilus:78.5,Thelichoris:96.5)HTU30:47.0)HTU26:56.0)HTU27:40.5)HTU29:30.5)HTU28:43.5,(((Limulus:76.0,(Vonones:86.0,(Pauroctonus:85.5,Thermobius:143.5)HTU23:49.0)HTU24:30.0)HTU19:34.5,(Amblypygid:87.0,(Americhernus:177.0,Aportus:162.0)HTU21:82.5)HTU20:65.0)HTU18:48.5,(Centruoides:79.0,Hadrurus:112.0)HTU22:86.0)HTU32:39.0)HTU25:39.5)HTU31:107.5)HTU17[1723.0];
(Alentus:106.5,(Chanbria:83.5,((Rhiphicephalus:150.5,(Artemia:164.5,(Mastigoproctus:94.0,(Gea:150.0,(Hypochilus:78.5,Thelichoris:96.5)HTU30:47.0)HTU26:56.0)HTU27:40.5)HTU29:30.5)HTU28:43.5,(((Limulus:76.0,(Vonones:86.0,(Pauroctonus:85.5,Thermobius:143.5)HTU23:49.0)HTU24:30.0)HTU19:34.5,(Amblypygid:87.0,(Americhernus:177.0,Aportus:162.0)HTU21:82.5)HTU20:65.0)HTU18:48.5,(Centruoides:79.0,Hadrurus:112.0)HTU22:86.0)HTU32:39.0)HTU25:39.5)HTU31:107.5)HTU17[1723.0];
(Alentus:106.5,(Chanbria:83.5,((Rhiphicephalus:150.5,(Artemia:164.5,(Mastigoproctus:94.0,(Gea:150.0,(Hypochilus:78.5,Thelichoris:96.5)HTU30:47.0)HTU26:56.0)HTU27:40.5)HTU29:30.5)HTU28:43.5,(((Limulus:76.0,(Vonones:86.0,(Pauroctonus:85.5,Thermobius:143.5)HTU23:49.0)HTU24:30.0)HTU19:34.5,(Amblypygid:87.0,(Americhernus:177.0,Aportus:162.0)HTU21:82.5)HTU20:65.0)HTU18:48.5,(Centruoides:79.0,Hadrurus:112.0)HTU22:86.0)HTU32:39.0)HTU25:39.5)HTU31:107.5)HTU17[1723.0];
